build.sbt

```diff
@@ -1,16 +1,25 @@
 import bintray.Keys.{repository, bintrayOrganization, bintray}
-
-scalaJSSettings
-
+import org.scalajs.sbtplugin.ScalaJSPlugin
+import org.scalajs.sbtplugin.ScalaJSPlugin.autoImport._
 organization               := "org.scalajs"
 
 name                       := "scala-parser-combinators"
 
-version                    := "1.0.2"
+version                    := "1.0.3-SNAPSHOT"
+
+scalaVersion               := "2.11.4"
+
+lazy val `scala-parser-combinators` = (project in file(".")).enablePlugins(ScalaJSPlugin)
```

Comment:
You can also just write `enablePlugins(ScalaJSPlugin)`, I believe. (New in 0.13.7, I think.)